GTG/gtk/browser/main_window.py

```diff
@@ -1381,40 +1381,40 @@ def get_selected_tree(self, refresh: bool = False):
         return self.req.get_tasks_tree(name=self.get_selected_pane(),
                                        refresh=refresh)
 
-    def get_selected_task(self, tv=None):
```

Comment:
Changes from this line downward are not strictly related to the change. I was just fixing some warnings and trying to improve readability. I can revert these changes if somebody insists.

I'm not sure `get_selected_task()` and `get_selected_tasks()` are needed now anyway, since we can do `self.get_pane().get_selection()`, but that's a problem for another day.